src/materials/ConcreteThermalMoisture.C

```diff
@@ -27,57 +27,50 @@ ConcreteThermalMoisture::validParams()
   params.addRequiredParam<std::string>(
       "type", "A string representing the Moose Object that is used to call this class");
 
-  // parameters for thermal properties calculations
-  MooseEnum thermal_conductivity_model("CONSTANT ASCE-1992 KODUR-2004 EUROCODE-2004", "CONSTANT");
-  MooseEnum thermal_capacity_model("CONSTANT ASCE-1992 KODUR-2004 EUROCODE-2004", "CONSTANT");
-  MooseEnum aggregate_type("Siliceous Carbonate", "Siliceous");
-
+  // available model for thermal and moisture transport
+  MooseEnum thermal_transport_model("CONSTANT ASCE-1992 KODUR-2004 EUROCODE-2004", "CONSTANT");
   params.addParam<MooseEnum>(
-      "thermal_conductivity_model", thermal_conductivity_model, "thermal conductivity models");
+      "thermal_transport_model", thermal_transport_model, "thermal conductivity models");
```

Comment:
I'd suggest these changes:
1) Change parameter name 'thermal_transport_model' to 'thermal_model'. I could be convinced to keep it how you have it if you prefer that, though.
2) Change parameter description to: 'Model for properties used in thermal transport'